detekt.yml

```diff
@@ -64,14 +64,14 @@ complexity:
     active: false
   LargeClass:
     active: true
-    threshold: 400
+    threshold: 150
   LongMethod:
-    active: false
-    threshold: 20
+    active: true
+    threshold: 60
```

Comment:
Optional: This is gonna be more than 60 😬. We used 80 in `Gulf` project. We can increase the threshold latter.